api-spec.md

```diff
@@ -125,33 +69,26 @@ Local Look-up
 #### Body Example:
 ```
 {
-  "device_id": "xxxxxx"
+  "fromDate": "timestamp",
+  "location": "city_name",
+  "getTraces": true,
 }
 ```
 #### Success Response:
 #### Code: `200 OK`
-##### Match Found:
+
 ```
  {
-   “found_diagnosed”: true,
-   “device”: [
-      { 
-        “device_id”: "xxxxxx",
-        "rssi": -70,
-        “encountered_at”: "timestamp"
+   "reports": [
+      {
+        "report": "cough, fever, headach",
+        "cenKeys": ["key_1", "key_2"],
+        "traces": [ {geo_hash } ]
```

Comment:
Considering all the comments above, I'd say this should be:

```suggestion
        "symptoms": "cough, fever, headache", // optional
        "tcnData": ["c432bf9538f248a8238543a89d08e098c89bb890"],
```

Reply:
New suggestion after reviewing `symptoms` part below: since we don't have to submit symptoms and we strive to minimize the size of payload, we should make it as simple as possible, so the final suggestion from me would be (for the whole response body):

```json
{
  "tcnReports": [
    "c432bf9538f248a8238543a89d08e098c89bb890",
    "78d78a7c8978e78a987f78e7a98f76328da987e987",
    "b321456787d57a854e878c431b8789e78c78a8875"
  ]
}
```

Reply:
okay, sounds good, do you have an idea of what the `cenKeys` array is for? Why did they originally propose to also upload them?

Reply:
@hyx131 Initially they did not have this smart CEN-generation algorithm that would allow to restore all CENs based on some seed data. So the reporter was expected to submit all CEN which they generated in the past. Now as they improved the algorithm, every client needs only initial seed data and can restore/re-generate all CENs (TCNs) from that data.